.github/workflows/docs.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
       - '4.0'
+      # temporary: allow to see GH Actions result on fork
```

Comment:
@Mogztter Please, remember to remove this after tested

Reply:
Ooops I didn't have time to remove it, I've opened: https://github.com/neo4j-contrib/neo4j-streams/pull/431